packages/kit/src/runtime/server/endpoint.js

```diff
@@ -27,7 +32,8 @@ function is_content_type_textual(content_type) {
 		type === 'text/plain' ||
 		type === 'application/json' ||
 		type === 'application/x-www-form-urlencoded' ||
-		type === 'multipart/form-data'
+		type === 'multipart/form-data' ||
+		type === 'text/event-stream'
```

Comment:
@kjmph I wasn't sure if this is necessary here - I was just reproducing what you previously had in adapter-utils.js - maybe it's not anymore since we check for async iterators in render_endpoint anyway? Could you have a look?

Reply:
Hello @jesseskinner, I don't believe this is necessary, for three reasons. A) you added the check that is the content type specified by the endpoint is not recognized, it will allow async iterators to progress in the response processing. B) the `is_content_type_textual` function is shared with the request processing on the POSTed body, this is undefined to have an `text/event_stream` being sent to the server. C) the `compressible` dependency will handle the actual question if there is a flush needed to get the compressor to emit the response body back to the client.

I am rebasing right now, and will remove this line.